alibi_detect/saving/_sklearn/saving.py

```diff
@@ -34,7 +35,7 @@ def save_model_config(model: BaseEstimator,
     filepath = base_path.joinpath(local_path)
     save_model(model, filepath=filepath, save_dir='model')
     cfg_model = {
-        'flavour': 'sklearn',
+        'flavour': Framework.SKLEARN.value,
```

Comment:
Inconsistency here where everywhere else `Framework.SKLEARN` was used, does it matter?

Reply:
See https://github.com/SeldonIO/alibi-detect/pull/650#discussion_r994464274.